reana_workflow_engine_yadage/cli.py

```diff
@@ -146,6 +146,7 @@ def run_yadage_workflow(
             updateinterval=5,
             loginterval=5,
             backend=cap_backend,
+            accept_metadir=True if "accept_metadir" in operational_options else False,
```

Comment:
I think this can be simplified:

```suggestion
            accept_metadir="accept_metadir" in operational_options,
```

Reply:
You are right! Much nicer :)